<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryView</class>
 <widget class="QWidget" name="HistoryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbxYear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>78</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPickPeriod">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick Period</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxMonth">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisplayHistory">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Display History</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="twHistoryDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>601</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabPomodoroHistory">
    <attribute name="title">
     <string>Pomodoro History</string>
    </attribute>
    <widget class="QListView" name="lvPomodoroHistory">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="todoHistory">
    <attribute name="title">
     <string>Todo History</string>
    </attribute>
    <widget class="QListView" name="lvTodoHistory">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOptional">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Optional</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
